public/src/bowling.js

```diff
@@ -0,0 +1,69 @@
+class Bowling {
+
+  constructor() {
+    this.MAX_FRAME_SCORE = 10;
+    this.MAX_FRAMES = 10;
+    this.firstBowl = true;
+    this.pins;
+    this.frame = 0;
+    this.frameScore;
+    this.gameScore = [[0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0]];
+  }
+  countScore(score) {
+    if(this.frame === this.MAX_FRAMES) {
+      this.firstBowl = true;
+      this.pins;
+      this.frame = 0;
+      this.frameScore;
+      this.gameScore = [[0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0], [0,0]];
+    } else {
+      this._countScoreHelper(score);
+    }
+    this.pins = score;
+  }
+  _countScoreHelper(score) {
+    if(score === this.MAX_FRAME_SCORE && this.firstBowl === true) {
+      this._countHelperWhenStrike(score);
+    } else {
+      this._countHelper(score);
+    }
+  }
+  getTotalScore() {
+    let gameScore = this.gameScore;
+    let score_modified = [];
+    gameScore.forEach(function(item, index) {
+      if(gameScore[index][0] === 10) {
+        let indexPlusOne = gameScore[index + 1].reduce((a, b) => a + b, 0);
+        let indexPlusTwo = gameScore[index + 2][0];
+        let score = gameScore[index + 1][0] === 10 ? indexPlusOne + indexPlusTwo : indexPlusOne;
+        let frameTotal = gameScore[index][0]
+        score_modified.push(frameTotal + score)
+      } else if((gameScore[index].reduce((a, b) => a + b, 0)) === 10) {
```

Comment:
It's super interesting seeing how you've used reduce function within your code, its a good way to achieve this i think!